package.json

```diff
@@ -138,9 +138,9 @@
   },
   "jest": {
     "moduleNameMapper": {
-      "^\/lib(.*)": "<rootDir>/lib/$1",
```

Comment:
No? This was a conflict in my branch, I thought I picked the development version, perhaps I chose the wrong one?